cassandra-mesos-framework/src/main/java/io/mesosphere/mesos/frameworks/cassandra/framework/Main.java

```diff
@@ -133,6 +137,17 @@ private static int _main() throws UnknownHostException {
         final String    defaultRack                 =                       Env.option("CASSANDRA_DEFAULT_RACK").or("RACK0");
         final String    defaultDc                   =                       Env.option("CASSANDRA_DEFAULT_DC").or("DC0");
 
+        final Map<String, String> dcOpts = Env.optionsStart("CASSANDRA_EXTERNAL_DC_", true);
+        final List<ExternalDc> externalDcs = newArrayList();
+        for (final String key: dcOpts.keySet()) {
+            externalDcs.add(
+                ExternalDc.newBuilder()
+                    .setName(key)
+                    .setUrl(dcOpts.get(key))
+                    .build()
+            );
+        }
```

Comment:
Please move lines 141:149 to their own (package private) method and write a unit test for this.